client/lib/keyboard.js

```diff
@@ -36,6 +44,15 @@ Mousetrap.bind(['down', 'up'], (evt, key) => {
   }
 });
 
+Mousetrap.bind('space', () => {
```

Comment:
BTW in Trello this shortcut works when hovering a card in the board view. Since we currently don’t have a good abstraction to implement that I won't ask you to do so, but could you please add a `// XXX comment` about it?